nx-X11/config/cf/linux.cf

```diff
@@ -677,6 +677,15 @@ XCOMM binutils:	(LinuxBinUtilsMajorVersion)
 # endif
 #endif /* AlphaArchitecture */
```

Comment:
> Non-ELF and ELF definitions for HPArchitecture are missing...

Yes, they are back now. Also the AsmDefines are as "before" when __hppa__ support was still present.